lib/SPIRV/libSPIRV/SPIRVStream.h

```diff
@@ -193,6 +193,7 @@ template <typename T>
 const SPIRVEncoder &operator<<(const SPIRVEncoder &O, T *P) {
   return O << P->getId();
 }
+template <> const SPIRVEncoder &operator<<(const SPIRVEncoder &O, SPIRVType *P);
```

Comment:
Can `P` be const too?
```suggestion
template <> const SPIRVEncoder &operator<<(const SPIRVEncoder &O, const SPIRVType *P);
```

Reply:
I've checked, and the answer is no. The translator uses non-const SPIRVType in the general template, so the `const SPIRVType` specialization won't ever be used.

Reply:
I see, thanks for checking anyway!